admin_manual/release_notes.rst

```diff
@@ -18,6 +19,113 @@ Release Notes
+
+New option to configure the language of mail notifications for public links
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Usually ownCloud renders mail notifications in the language of the recipients, when they are known. For the `recently improved feature <https://doc.owncloud.com/server/10.0/admin_manual/release_notes.html#personal-note-for-public-link-mail-notification>`_ to send public links with a personal note directly from the user interface the recipients' language can't be determined automatically it just knows the recipients' mail addresses. ownCloud therefore uses the language of the user who sent the notification which can have the drawback that recipients can't understand them.
+This is still the default behavior but administrators can now change it via a dropdown menu *Language used for public mail notifications for shared files* in the settings *Sharing* section.
+
+Theming changes
+~~~~~~~~~~~~~~
+
+Mail templates for share notifications do not strip linebreaks from the personal note anymore. This affects the HTML (``core/templates/mail.php``) and plain text (``core/templates/altmail.php``) mail templates. The default templates shipped with ownCloud Server 10.0.10 have been modified to accomodate to these changes. If your custom theme overrides these templates, you have to follow up with the changes:
+
+- Replace the following line of the HTML template
+``p($l->t("Personal note from the sender: %s.", [$_['personal_note']]));``
+
+with
+
+``print_unescaped($l->t("Personal note from the sender: <br> %s.", $_['personal_note']));``.
+
+- Replace the following of the plain text template
+
+``print_unescaped($l->t("Personal note from the sender: %s.", [$_['personal_note']]));``
+
+with
+
+``print_unescaped($l->t("Personal note from the sender: \n %s.", $_['personal_note']));``.
+
+Other notable changes
+~~~~~~~~~~~~~~~~~~~~~
+- Allow automated SSL certificate verifications for other CAs than Let's Encrypt. See `#31858 <https://github.com/owncloud/core/issues/31858>`_ for further details.
+-  "/" and "%" are now valid characters in group names. See `#31109 <https://github.com/owncloud/core/issues/31109>`_ for further details.
+- New audit events for login action with token or Apache. See `_#31985 <https://github.com/owncloud/core/issues/31985>`_ for further details.
+- Log entries for user quota exceedance: Loglevel changed to "debug" (Insufficient storage exception is now logged with "debug" log level).
```

Comment:
s/for user quota exceedance/for exceeding user quota/

but ``exceedance`` should be a word ;) We could add it to the dictionary.

Reply:
I even looked it up! Seems to be a word https://www.dictionary.com/browse/exceedance :D 
Anyway, changed it to your proposal, thx